webrtc.html

```diff
@@ -1313,8 +1308,55 @@
               </li>
 
               <li>
-                <p>If the candidate is successfully applied, resolve
-                <var>p</var> with <code>undefined</code>.</p>
+                <p>In parallel, start the process to apply
+                <var>candidate</var>.</p>
+
+                <div class="note">We probably want to define the process to
+                'apply an ICE candidate' in JSEP and link to it above.</div>
```

Comment:
We could make a generic link to JSEP, but we can't use the nice feature and point to a specific section (that doesn't exist yet).